src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -386,7 +386,24 @@ bool AsyncRPCOperation_sendmany::main_impl() {
             expsk = sk.expsk;
             from = expsk.full_viewing_key();
         } else {
-            // TODO: Set "from" to something!
+            // Sending from a t-address, which we don't have an ovk for. Instead, use
+            // the FVK for Sapling account 0 (even if the user has not generated it).
+
+            HDSeed seed;
+            if (!pwalletMain->GetHDSeed(seed))
+                throw JSONRPCError(
+                    RPC_WALLET_ERROR,
+                    "CWallet::GenerateNewSaplingZKey(): HD seed not found");
+
+            auto m = libzcash::SaplingExtendedSpendingKey::Master(seed);
```

Comment:
Is there some code duplication here? Guessing there's somewhere in the code where we derive the 0 account. Also perhaps add a reference to zip 32/short explanation to what account 0 means; btw also in zip 32 you currently don't use this terminology - that account i means `m_Sapling / purpose' / coin_type'/i`. Perhaps add it there, i.e. add a sentence 'We sometimes use informally refer to the path `m_Sapling / purpose' / coin_type'/i` as the  i'th account'

Reply:
There is kind-of duplication here from [the key generation code](https://github.com/zcash/zcash/blob/master/src/wallet/wallet.cpp#L118-L133), but not quite because here we know exactly which account we want, whereas there we are searching for an unused one.

We define `account` in ZIP 32 [here](https://github.com/zcash/zips/pull/157/files#diff-395ced896b8bf805b190247312c2fc4eR310) to have the same interpretation as in BIP 44.

`DefaultAddress()` (both in the implementation and the ZIP) refers to the default diversified `PaymentAddress` for a given extended key (which could be anywhere along any path), not the default account for the specific ZIP 32 Sapling path.